audio/m4a.py

```
@@ -0,0 +1,35 @@
+        print(f"Converting: {filename} → {output_name}")
+
+        cmd = [
+            FFMPEG_EXE,
+            "-y",
+            "-i", input_path,
+            "-ac", "1",
+            "-ar", "16000",
+            output_path
+        ]
+
+        result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)